schematools/importer/__init__.py

```diff
@@ -41,13 +41,28 @@
 }
 
 
+def numeric_datatype_scale(scale_=None):
+    if scale_:
```

Comment:
Het is veiliger om tegen None te testen dus `if scale_ is not None`. Wat zou er moeten gebeuren als een `scale_ = 0.0` wordt gebruikt?

Reply:
Ik heb de logica gewijzigd, nu wordt gekeken naar membership van number en tevens extra tests voor invalide invoer zoals 0.0 of 9.1234 etc.